database/rrdlabels.c

```diff
@@ -159,6 +159,31 @@ int label_list_contains(struct label *head, struct label *check)
     return label_list_contains_key(head, check->key, check->key_hash);
 }
 
+struct label *label_list_lookup_keylist(struct label *head, char *key)
+{
+    SIMPLE_PATTERN *pattern = NULL;
+
+    pattern = simple_pattern_create(key, ",|\t\r\n\f\v", SIMPLE_PATTERN_EXACT);
+
+    if (!pattern)
```

Comment:
The function `simple_pattern_create` calls (`mallocz`)[https://github.com/netdata/netdata/blob/15a81d70d6ffa7b950c797ca8f1058b97d32495d/libnetdata/simple_pattern/simple_pattern.c#L92] that will shutdown everything when there is not memory available, so this test is not necessary.

Reply:
I have removed it. There are others checks in place to make sure this works fine even without this condition.